src/main/java/hudson/plugins/audit_trail/LogFileAuditLogger.java

```diff
@@ -30,33 +48,143 @@ public class LogFileAuditLogger extends AuditLogger {
 
     private static final Logger LOGGER = Logger.getLogger(LogFileAuditLogger.class.getName());
     static final String DEFAULT_LOG_SEPARATOR=" ";
+    protected static final String DAILY_ROTATING_FILE_REGEX_PATTERN = "-[0-9]{4}-[0-9]{2}-[0-9]{2}" + ".*" + "(?<!lck)$";
```

Comment:
nit: this can be package protected:

```suggestion
   static final String DAILY_ROTATING_FILE_REGEX_PATTERN = "-[0-9]{4}-[0-9]{2}-[0-9]{2}" + ".*" + "(?<!lck)$";
```